GoogleForCreators/web-stories-wp
# Editor: Updated close icon and fixed input text overlap

## Summary

This PR updates the text input used in the Editor.  It updates the clear text icon, and prevents the text from being clipped by the icon.

## User-facing changes

The text input component used in the "Link" panel and the "Accessibility" panel has been updated.

## Testing Instructions

1.) Open a story in the Editor.
2.) Add an image to the page.
3.) Click on the image and type in really long text into the "Link" and/or "Accessibility" panel and notice that the "clear text" icon is a nice big X, and notice that the input text does _not_ get overlapped by the icon (it gets clipped).

Fixes #1874 

## Screenshot 

![long_text](https://user-images.githubusercontent.com/40646372/90940301-22313700-e3c3-11ea-9790-fe72d89d80d3.gif)
